specificatie/klanten/openapi.yaml

```diff
@@ -1077,97 +1123,106 @@ components:
           nullable: true
       discriminator:
         propertyName: subjectType
-    VerblijfsAdres:
-      title: Verblijfsadres
+    Binnenlandsadres:
```

Comment:
@HenriKorver We hebben het daar al kort over gehad maar je zou nog kunnen overwegen om hier 'aanduidingBijHuisnummer' aan toe kunnen voegen. Bij HC BRP Personen Bevragen kennen we die ook en daar is het een waardetabel. Deze kan de waarden 'tegenover' en 'bij' hebben.

Reply:
Zie #163